CMakeLists.txt

```diff
@@ -295,7 +295,7 @@ if(cmake_build_type_lower MATCHES "coverage")
         '/usr/*'
         ${CMAKE_SOURCE_DIR}'/3rd-party/*'
         ${CMAKE_SOURCE_DIR}'/tests/*'
-        ${CMAKE_BINARY_DIR}
+        ${CMAKE_BINARY_DIR}'/*'
```

Comment:
Makes sense to me even though it does not change the behavior.

Reply:
Doesn't it change behavior for you? It does for me, locally. It may have to do with tool versions...

Reply:
> I suppose Codecov knows to exclude anything that is not part of the repo, which would explain why its results are not impacted.

I am talking about this one. Meaning that Codecov does not count anything in to calculate the code coverage.

Reply:
Ah OK. Right so this is meant to fix coverage in general, not just codecov.